src/nanoemoji/colr_to_svg.py

```diff
@@ -407,7 +399,13 @@ def colr_to_svg(
     rounding_ndigits: Optional[int] = None,
 ) -> Dict[str, SVG]:
     """For testing only, don't use for real!"""
-    assert len(ttfont["CPAL"].palettes) == 1, "We assume one palette"
+    num_palettes = len(ttfont["CPAL"].palettes)
+    if num_palettes > 1:
+        logging.warning(
+            "Multiple CPAL palettes are not supported! Only using the first"
+        )
+    elif num_palettes < 1:
```

Comment:
Optional, control flow would be simpler if you just replace the original assert with > 0, must have at least one palette.

Hm. I'm tempted to suggest it should be OK to not have any palettes, then the only color you can use is foreground.

Reply:
> I'm tempted to suggest it should be OK to not have any palettes, then the only color you can use is foreground.

it took me reading the sentence three times to realise it was a joke

Reply:
actually, you _are_ serious, you mean COLR without CPAL..

Reply:
note to self, a CPAL the table still is required for COLR:

https://docs.microsoft.com/en-us/typography/opentype/spec/colr
> If the COLR table is present in a font but no CPAL table exists, then the COLR table is ignored.

but I suppose it could well contain no palettes if only foreground is used

Reply:
> If the COLR table is present in a font but no CPAL table exists, then the COLR table is ignored.

I want to change that too, I think COLR without CPAL is fine. But we can leave it until COLRv2 to be consistent with the spec if you prefer.

Reply:
ok, I'll keep a logging.warning for len(palettes) > 1, but move the check for CPAL presence and len(palettes) > 0 closer to where it's needed (inside `_color` function where it looks up a color in the CPAL palette) and will no longer assert that a non empty CPAL exists if all colors are foregrounds